be/src/test/java/com/example/be/comment/CommentRepositoryTest.java

```diff
@@ -0,0 +1,105 @@
+package com.example.be.comment;
+
+import com.example.be.issue.IssueRepository;
+import com.example.be.issue.dto.IssueCreateFormDTO;
+import com.example.be.user.User;
+import com.example.be.user.UserRepository;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Transactional
+@SpringBootTest
+class CommentRepositoryTest {
+
+    @Autowired
+    private CommentRepository commentRepository;
+
+    @Autowired
+    private IssueRepository issueRepository;
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Test
+    @DisplayName("이슈 번호와, 유저 아이디, 댓글 내용을 파라미터로 받아 댓글을 저장할 수 있다.")
+    void save() {
+        // given
+        User user = userRepository.save(new User("hyun", "codesquad", "hyun", "mock.img"));
+        IssueCreateFormDTO issueCreateFormDTO = new IssueCreateFormDTO("테스트", "내용", null, null, null);
+        issueCreateFormDTO.setUserId(user.getId());
+
+        int issueNumber = issueRepository.save(issueCreateFormDTO);
+
+        String contents = "댓글 내용이당";
+
+        // when
+        commentRepository.save(issueNumber, user.getId(), contents);
+
+        // then
+        List<Comment> comments = StreamSupport.stream(commentRepository.findAll().spliterator(), false)
+                .collect(Collectors.toList());
```

Comment:
```suggestion
        List<Comment> comments = commentRepository.findAll().stream()
                .map() // map이나 filter 적용
                .collect(Collectors.toList());
```
이렇게 작성하는법도 있는데 어떻게 생각하세요?

Reply:
훨씬 직관적이고 보기 깔끔한 것 같습니다! 
해당 내용 참고하여 리팩토링 해보겠습니다!